### ISSUE TITLE
Configuration Map syntax throws fatal error

### ISSUE THREAD

**Issue by luetkemj** [sasstools/sass-lint#994]
What version of Sass Lint are you using? 1.10.2

Please include any relevant parts of your configuration
Using cssModules
Webpack 1.14.0
style-loader 0.13.1
css-loader 0.26.1

What did you do? Please include the actual source code causing the issue.
Wrote this mixin:
```
@mixin font($font-size: $medium, $font-family: $sans-serif, $font-weight: $normal) {
  font-family: $font-family;
  font-size: $font-size;
  font-weight: $font-weight;
}
```
Used like this:
```
.name {
  @include font((font-size: $large)...);
  color: $swatch07;
  margin: 0 0 4px;
}
```
What did you expect to happen?
I expected the linter to either not throw an error or call out a rule I could modify to allow the syntax.

What actually happened? Please include any error messages given to you by Sass Lint.
The code compiles and looks great in the UI but the linter throws this error and fails my build:
![screen shot 2017-01-13 at 3 57 24 pm](https://cloud.githubusercontent.com/assets/925980/21947124/281df4e6-d9a9-11e6-8401-5de2fb427e92.png)

The syntax used is described in this article: https://www.sitepoint.com/using-sass-maps/

If you're using an IDE plugin have you tried the CLI too?
Yes. My build fails. NPM script is 
```
"lint:sass-lint": "sass-lint -c ./.sass-lint.yml './app/**/*.scss' -v"
```

**Comment by clement-escolano** [comment]
Hello,

I have the same error with the same package version but completely different environment. I have no config file. When I launch `node node_modules/sass-lint/bin/sass-lint.js`, I don't have the errors displayed but I see the error message beginning with `throw new exception.SassLintFailureError`.

I don't know if I am doing something wrong but I am not able to use sass-lint right now :( 

Thank you !
